caffe2/proto/torch.proto

```diff
@@ -174,6 +177,9 @@ message NodeProto {
 
   // A human-readable documentation for this node. Markdown is allowed.
   optional string doc_string = 6;
+
```

Comment:
Peter is making a change to store the schema (just a string) in the doc_string for now. We could add an additional scheme field here for that.